<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncidenceGeneralEditor</class>
 <widget class="QWidget" name="IncidenceGeneralEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the title</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets the Title of this event or to-do.</string>
     </property>
     <property name="text">
      <string extracomment="@label">T&amp;itle:</string>
     </property>
     <property name="buddy">
      <cstring>mSummaryEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="KLineEdit" name="mSummaryEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the title</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets the Title of this event or to-do.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string extracomment="@label">&amp;Location:</string>
     </property>
     <property name="buddy">
      <cstring>mLocationEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="KLineEdit" name="mLocationEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the location</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets where the event or to-do will take place.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string extracomment="@label">Categories:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="KPIM::KCheckComboBox" name="mCategoryCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the secrecy level</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets whether the access to this event or to-do is restricted. Please note that KOrganizer currently does not use this setting, so the implementation of the restrictions will depend on the groupware server. This means that events or to-dos marked as private or confidential may be visible to others.</string>
     </property>
     <property name="text">
      <string>Access:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="mSecrecyCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Set the secrecy level</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Sets whether the access to this event or to-do is restricted. Please note that KOrganizer currently does not use this setting, so the implementation of the restrictions will depend on the groupware server. This means that events or to-dos marked as private or confidential may be visible to others.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPIM::KCheckComboBox</class>
   <extends>KComboBox</extends>
   <header>libkdepim/kcheckcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
